src/Compilers/CSharp/Portable/Symbols/Source/SourceMemberContainerSymbol.cs

```diff
@@ -3063,8 +3066,12 @@ private void AddSynthesizedConstructorsIfNecessary(ArrayBuilder<Symbol> members,
                     switch (method.MethodKind)
                     {
                         case MethodKind.Constructor:
-                            hasInstanceConstructor = true;
-                            hasParameterlessInstanceConstructor = hasParameterlessInstanceConstructor || method.ParameterCount == 0;
+                            // Ignore the record copy constructor
+                            if (!(method is SynthesizedRecordCopyCtor))
```

Comment:
Can we check 'method.IsImplicitlyDeclared' here instead?

Reply:
That's true for all SynthesizedInstanceMethodSymbols, including the record primary constructor, unfortunately